scripts/demo.js

```diff
@@ -1,51 +1,29 @@
 import path from "path";
 import fs from "fs";
-import fse from "fs-extra";
-import { section, generateCommand } from "../utils.js";
-import { execSync } from "child_process";
-import { configurePython, execOptions } from "./utils/environment.js";
+import { section } from "../utils.js";
+import fetch from "node-fetch";
+import config from "../config.js";
+import unzipper from "unzipper";
 
-export function createDemo(dir, yaml) {
-  const options = Object.assign(execOptions, {
-    cwd: path.dirname(dir)
-  });
+async function downloadAndExtract(url, target) {
+  const rnResponse = await fetch(url);
+  if (rnResponse.ok && rnResponse.body) {
+    rnResponse.body.pipe(unzipper.Extract({ path: target }));
```

Comment:
nit: this won't have any side effects? I mean, if the .pipe fails somehow, we are not catching any exception and throwing the error to the User.

Reply:
Thanks for the comment! Good point... The idea here is that these errors will bubble up to the main dispatch call and the error will be sent to sentry. That's why I also throw the error in the if/else instead of just showing an error message.

But I think we could improve it a bit since this makes the user see an unhandled async error message from Node. I made a small change in 9db1bd4 so that the error messages are routed to sentry and also "parsed". Improvements to the parsing function will probably require some product input, so I will leave it as is for now!